SpringProject/src/main/java/com/gradproject2019/Controller.java

```diff
@@ -1,13 +1,20 @@
 package com.gradproject2019;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 
 @org.springframework.stereotype.Controller
-@RequestMapping(path="/all")
+//@RequestMapping(path="/all")
 public class Controller {
 
     @Autowired
     private ConferenceRepository conferenceRepository;
 
-}
+    @GetMapping(path="/ListOfConferences")
+    public @ResponseBody Iterable<Conference> getAllUsers(){
+        return conferenceRepository.findAll();
```

Comment:
You will need a service layer between the controller and the repository layer. You can do that on this branch and do both tickets at once.

Reply:
(Not best practise but for the sake of expediting things)